docs/docson/build-schema.json

```diff
@@ -185,6 +185,33 @@
             ],
             "description": "can be either a single sourceItem or an array of sourceItems, sourceItem can be either a single string(non nested directory) or an detailed object config"
         },
+        "targetItem": {
+          "oneOf": [
+              {
+                "type": "string",
+                "enum": [
+                  "js"
+                ],
+                "description": "JS target compiled using BuckleScript"
+              },
+              {
+                "type": "object",
+                "properties": {
+                  "kind": {
+                    "enum": [
+                      "native",
+                      "bytecode"
+                    ],
+                    "description": "The compiler to use for the target"
+                  },
+                  "entrypoint": {
+                    "type": "string",
+                    "description": "Path to file used as entrypoint for this target"
```

Comment:
"entry-point" is just used to understand how to build. If you're making a library it'll be used for an example or tests, for example. It doesn't get used when someone uses the project as a library. I can update the description.